src/js/components/tablero/GraphCard.jsx

```diff
@@ -48,7 +60,20 @@ const GraphCard = SortableElement(({
         }
         <DragHandle />
       </div>
-      <div className="contentCard">{graph}</div>
+      <div className="contentCard">
+        <div className="dataFilter">
+          <select
+            className={filter ? 'customSelect' : 'customSelect disabled'}
+            onChange={e => reloadIndicator(cardMethod, cardType, cardTitle, cardLink, cardId, `querySize=${e.target.value}`)}
+            disabled={!filter}
+          >
+            <option value="6">Last 6 Months</option>
```

Comment:
I quess this could be added to localization later and translation would be used.